PRChecker/Shared/PullRequestCell.swift

```diff
@@ -69,18 +97,107 @@ private struct BranchLabel: View {
     let labelText: String
     let type: BranchType
 
+    var body: some View {
+        Tag(text: labelText, color: type.color)
+    }
+}
+
+// MARK: - Body
+
+private struct ContentBody: View {
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            // User
+            HStack(alignment: .top) {
+                Image(systemName: "person")
+                    .scaledToFit()
+                    .foregroundColor(Colors.darkGray)
+                Text("User-1234567")
```

Comment:
FYI: You can actually input `Image(systemName:)` in `Text` so that it grows with Dynamic font